packages/codegen-ui-react/lib/__tests__/forms/validation.test.ts

```diff
@@ -157,77 +157,105 @@ describe('validateField tests', () => {
       validateField(3, [{ type: ValidationTypes.EQUAL_TO_NUM, numValues: [4, 5, 6], validationMessage: 'test' }]),
     ).toEqual({ hasError: true, errorMessage: 'test' });
   });
-  it('should validate BE_AFTER type', () => {
-    const startDate = new Date().toDateString();
-    const endDate1 = new Date('2021-01-09').toDateString();
-    const endDate2 = new Date('3000-01-09').toDateString();
-    expect(
-      validateField(startDate, [{ type: ValidationTypes.BE_AFTER, strValues: [endDate1], validationMessage: '' }]),
-    ).toEqual({ hasError: false });
-    expect(
-      validateField(startDate, [{ type: ValidationTypes.BE_AFTER, strValues: [endDate2], validationMessage: '' }]),
-    ).toEqual({ hasError: true, errorMessage: `The value must be after ${endDate2}` });
-    expect(
-      validateField(startDate, [{ type: ValidationTypes.BE_AFTER, strValues: [''], validationMessage: 'test' }]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
+  describe('DateTime tests', () => {
+    it('should parse Date and timestamp', () => {
+      const time = Date.now().toString();
+      const timestamp = parseDateValidator(time);
+      const invalidTimestamp = parseDateValidator('1232131asdfasf123');
+      const ddMMYYYY = parseDateValidator('01 Jan 2022');
```

Comment:
These should be separate tests or at least use `.each`

Reply:
Yeah thats better. Will fix in another pr